ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/utils/stringUtils.ts

```diff
@@ -0,0 +1,16 @@
+import { ClusterType } from 'types/cluster.proto';
+import { ensureExhaustive } from 'utils/type.utils';
+
+export function displayClusterType(type: ClusterType): string {
+    switch (type) {
+        case 'GENERIC_CLUSTER':
+            return 'Generic';
```

Comment:
You might double check with backend coworkers whether `'GENERIC_CLUSTER'` is possible.

For comparison:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Clusters/cluster.helpers.tsx#L37-L41

It sticks in my mind the absence of that constant value came up, and we learned it was for Docker Swarm clusters in the early days of the product. When in doubt, better safe than sorry, though.

Reply:
Agreed that I think this is effectively deprecated at this point. It is still included in the API documentation though, so I figured I would cover it for completeness.